frontend/src/metabase/lib/core.js

```diff
@@ -9,78 +11,78 @@ export const user_roles = [{
 }];
 
 export const field_special_types = [{
-    'id': 'id',
+    'id': TYPE.PK,
```

Comment:
like this refactor. not a huge fan of bare strings with specific types all over the place.